app/test_main.py

```diff
@@ -1,3 +1,23 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, counted_ages",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
+    ]
+)
```

Comment:
The test cases provided cover a good range of scenarios for typical inputs. However, according to the checklist, you should also test edge cases such as negative numbers, zero, or very large numbers to ensure the function handles these gracefully. Consider adding such test cases to improve the robustness of your tests.